### PR TITLE
Exterminate all non-canonical EIP links

### PR DESCRIPTION
**Pull request by Pandapip1**
## Description

Some links to EIPs used the GitHub PR, legacy issue, or GitHub-hosted file. This PR updates those to use the canonical links.

Translations have **not** been updated, due to the process being time consuming.

## Related Issue

None

### CODE REVIEW

File: src/content/developers/docs/consensus-mechanisms/pow/mining-algorithms/ethash/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ ACCESSES = 64                     # number of accesses in hashimoto loop
 Ethereum's development coincided with the development of the SHA3 standard, and the
 standards process made a late change in the padding of the finalized hash algorithm, so that Ethereum's
 "sha3_256" and "sha3_512" hashes are not standard sha3 hashes, but a variant often referred
-to as "Keccak-256" and "Keccak-512" in other contexts. See discussion, e.g. [here](https://github.com/ethereum/EIPs/issues/59), [here](http://ethereum.stackexchange.com/questions/550/which-cryptographic-hash-function-does-ethereum-use), or [here](http://bitcoin.stackexchange.com/questions/42055/what-is-the-approach-to-calculate-an-ethereum-address-from-a-256-bit-private-key/42057#42057).
+to as "Keccak-256" and "Keccak-512" in other contexts. See discussion, e.g. [here](https://eips.ethereum.org/EIPS/1803), [here](http://ethereum.stackexchange.com/questions/550/which-cryptographic-hash-function-does-ethereum-use), or [here](http://bitcoin.stackexchange.com/questions/42055/what-is-the-approach-to-calculate-an-ethereum-address-from-a-256-bit-private-key/42057#42057).
```
Comment (Pandapip1) [inline src/content/developers/docs/consensus-mechanisms/pow/mining-algorithms/ethash/index.md:50]:
```suggestion
to as "Keccak-256" and "Keccak-512" in other contexts. See discussion, e.g. [here](https://eips.ethereum.org/EIPS-1803), [here](http://ethereum.stackexchange.com/questions/550/which-cryptographic-hash-function-does-ethereum-use), or [here](http://bitcoin.stackexchange.com/questions/42055/what-is-the-approach-to-calculate-an-ethereum-address-from-a-256-bit-private-key/42057#42057).
```

File: src/content/eips/index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Anyone within the Ethereum community has the ability to create an EIP. Guideline
 
 ## Why do EIPs matter? {#why-do-eips-matter}
 
-EIPs play a central role in how changes happen and are documented on Ethereum. They are the way for people to propose, debate and adopt changes. There are [different types of EIPs](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-types), including core EIPs for low-level protocol changes that affect consensus and require a network upgrade like [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559), and ERCs for application standards like [EIP-20](https://eips.ethereum.org/EIPS/eip-20) and [EIP-721](https://eips.ethereum.org/EIPS/eip-721).
+EIPs play a central role in how changes happen and are documented on Ethereum. They are the way for people to propose, debate and adopt changes. There are [different types of EIPs](https://eips.ethereum.org/EIPS/eip-1.md#eip-types), including core EIPs for low-level protocol changes that affect consensus and require a network upgrade like [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559), and ERCs for application standards like [EIP-20](https://eips.ethereum.org/EIPS/eip-20) and [EIP-721](https://eips.ethereum.org/EIPS/eip-721).
```
Comment (Pandapip1) [inline src/content/eips/index.md:17]:
```suggestion
EIPs play a central role in how changes happen and are documented on Ethereum. They are the way for people to propose, debate, and adopt changes. There are [different types of EIPs](https://eips.ethereum.org/EIPS/eip-1#eip-types), including core EIPs for low-level protocol changes that affect consensus and require a network upgrade like [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559), and ERCs for application standards like [EIP-20](https://eips.ethereum.org/EIPS/eip-20) and [EIP-721](https://eips.ethereum.org/EIPS/eip-721).
```

**Review by Pandapip1 — COMMENTED** [review]
Self-review: typo fixes

**Comment by Pandapip1** [comment]
👋 anyone willing to review this?

**Comment by corwintines** [comment]
Ill go through and fix all these broken links, but thanks for getting the ball rolling @Pandapip1!

**Review by corwintines — APPROVED** [review]

**Pull request merged**